src/vs/workbench/parts/extensions/electron-browser/extensionEditor.ts

```diff
@@ -260,11 +265,12 @@ export class ExtensionEditor extends BaseEditor {
 		switch (id) {
 			case NavbarSection.Readme: return this.openReadme(extension);
 			case NavbarSection.Contributions: return this.openContributions(extension);
+			case NavbarSection.Changelog: return this.openChangelog(extension);
 		}
 	}
 
-	private openReadme(extension: IExtension) {
-		return this.loadContents(() => this.extensionReadme.get()
+	private openMarkdown(extension: IExtension, content: TPromise<string>, noContentCopy: string) {
```

Comment:
Yeah perfect. We don't need that `extension` argument any way, right? I must've left it behind somehow. You can remove that.